automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -94,37 +95,21 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
       val role = "bigquery.jobUser"
 
       val user: Credentials = UserPool.chooseStudent
-      val userToken: AuthToken = user.makeAuthToken()
-
-      // user is not a project owner
-      val errorMsg1 = "You must be a project owner"
-
-      withBillingProject("auto-goog-role") { projectName =>
-        val addEx = intercept[APIException] {
-          Orchestration.billing.addGoogleRoleToBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        addEx.getMessage should include(errorMsg1)
-
-        val removeEx = intercept[APIException] {
-          Orchestration.billing.removeGoogleRoleFromBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        removeEx.getMessage should include(errorMsg1)
-      }(ownerToken)
-
-      // owner is not an owner of this project
-      val errorMsg2 = "The caller does not have permission"
 
-      val otherProject = "broad-dsde-dev"
```

Comment:
I was running tests in dev with a user who actually *was* a broad-dsde-dev owner.  Oops.  These are not actually two different cases.